src/FileOnQ.Imaging.Raw/UnpackedImage.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+
+namespace FileOnQ.Imaging.Raw
+{
+	unsafe class UnpackedImage : IUnpackedImage
+	{
+		RawImageData data;
+		IImageProcessor processor;
+
+		internal UnpackedImage(IntPtr libraw)
+		{
+			data = new RawImageData
+			{
+				LibRawData = libraw
+			};
+		}
+
+		public void Process(IImageProcessor newProcessor)
+		{
+			if (processor != null)
+				processor.Dispose();
```

Comment:
This is the safety check that was documented in the original PR description. If we already have a processor we need to `Dispose()` then we can continue. This should prevent any weird memory situations